kafka-producers/templates/deployment.yaml

```diff
@@ -0,0 +1,68 @@
+{{- range $producer, $values := .Values.producers }}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ printf "%s-%s" $.Release.Name $producer | trunc 63 | trimSuffix "-" }}
+  labels:
+    helm.sh/chart: {{ include "kafka-producers.chart" $ }}
+    app.kubernetes.io/instance: {{ $.Release.Name }}
+  {{- include "multicast.annotation" . | nindent 2 }}
```

Comment:
not sure if you need to put the multicast annotation here, but it probably doesn't hurt

Reply:
I think all the examples I saw duplicated the annotations across the spec, so I just went with the flow.